src/manipulation.js

```diff
@@ -45,7 +45,6 @@ var
 
 	// checked="checked" or checked
 	rchecked = /checked\s*(?:[^=]|=\s*.checked.)/i,
-	rscriptTypeMasked = /^true\/(.*)/,
```

Comment:
The changes in this file seem unrelated to the PR?

Reply:
Indeed. I didn't see that before. Nice catch!